### ISSUE TITLE
[Bug]: Accessibility - SVG icons being announced by screenreader

### ISSUE THREAD

**Issue by janehughes** [ONSdigital/design-system#3583]
### Describe the bug

There are several instances of decorative SVG icons being announced by the screenreader, across a few different components

1. Header - menu toggle button arrow icon
2. Breadcrumb arrow divider icons
3. Details panel - show/hide toggle arrow icon
4. Accordion arrow icon

### Expected behaviour / outcome

Any icons used for decorative purposes should have an `aria-hidden` attribute so they are ignored by screenreaders, otherwise they create very verbose announcements which are not helpful and could be frustrating for users, especially in components that have repeated icons (e.g. breadcrumbs, accordions)

### List the steps necessary to reproduce the bug

Review example components as listed above or preview page which includes components e.g. https://ons-beta.torchbox.dev/dac-exemplar/inflation-and-price-indices-dac/retail-sales-index-rsi-qmi/

### What version of the ONS Design System are you using?

72.9.1

### What device(s) are you using?

Desktop

### What operating system(s) are you using?

macOS

### What browser(s) are you using?

Chrome

**Comment by MagdalenaLarge** [comment]
PR: https://github.com/ONSdigital/design-system/pull/3592
